libs/core/src/components/badge/badge.ts

```diff
@@ -0,0 +1,74 @@
+import { html } from 'lit/static-html.js'
+import { unsafeCSS } from 'lit'
+import { gdsCustomElement } from '../../utils/helpers/custom-element-scoping'
+import { GdsElement } from '../../gds-element'
+import { tokens } from '../../tokens.style'
+import { styleExpressionProperty } from '../../utils/decorators/style-expression-property'
+
+import BadgeCSS from './badge.style.css?inline'
```

Comment:
Here, I would
* rename this file to `badge.style.ts` to align with other components
* remove the `inline` query, since that is for static files
* remove `unsafeCSS` below, since we're using the `css` template tag from Lit